src/config.rs

```diff
@@ -2416,6 +2423,7 @@ impl Default for CdcConfig {
             // is more than 500MB/s, so 128MB/s is enough.
             incremental_scan_speed_limit: ReadableSize::mb(128),
             incremental_scan_ts_filter_ratio: 0.2,
+            tso_worker_threads: 1,
```

Comment:
better to add a range limitation for its value.

Reply:
How to change it on the Cloud?

Reply:
Do you mean dynamically change it in TiDB? It's not available now.